test/modules/TestResultCoordinator/Reports/DeploymentTestReportGenerator.cs

```diff
@@ -106,7 +106,7 @@ public async Task<ITestResultReport> CreateReportAsync()
 
             while (hasExpectedResult)
             {
-                unmatchedResults.Add(this.ExpectedTestResults.Current);
+                TestReportUtil.EnqueueAndEnforceFixedSize(unmatchedResults, this.ExpectedTestResults.Current, Settings.Current.UnmatchedResultsMaxSize);
```

Comment:
nit: seems the line swap looks weird to me; either break all arguments into its own line or all in the same line.

Reply:
I think that is just the rendering. It's on the same line.